brainstormforce/ultimate-addons-for-gutenberg
# Fix: Console error in Color Control due to which block not rendering.

### Description

- Runtime task need to fix on P1

### Screenshots

- NA

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop a new block should not throw any error.
- Block should be rendered properly, Adding a Color should work as expected.
- Check BoxShadow Preset once.

### Checklist:
